auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue

```diff
@@ -5,24 +5,36 @@
         <label><strong>My List ({{ entityCount }})</strong></label>
       </div>
       <v-data-table
-          id="affiliated-entity-table"
-          :headers="getHeaders"
-          :items="businesses"
-          :items-per-page="1000"
-          fixed-header
-          height="30rem"
-          hide-default-footer
+        id="affiliated-entity-table"
+        :headers="getHeaders"
+        :items="businesses"
+        :items-per-page="1000"
+        :height="getMaxHeight"
+        fixed-header
+        hide-default-footer
       >
         <template v-slot:item="{ item, index }">
           <tr>
-            <td class="copy-normal">{{item.name}}</td>
-            <td v-if="showCol(headers[1].text)" class="copy-normal">{{item.number}}</td>
-            <td v-if="showCol(headers[2].text)" class="copy-normal">{{item.type}}</td>
-            <td v-if="showCol(headers[3].text)" class="copy-normal">{{item.status}}</td>
-            <td v-if="showCol(headers[4].text) && isPremiumAccount" class="copy-normal">{{item.folio}}</td>
-            <td v-if="showCol(headers[5].text)" class="copy-normal">{{item.lastModified}}</td>
-            <td v-if="showCol(headers[6].text)" class="copy-normal">{{item.modifiedBy}}</td>
-            <td class="copy-normal action-cell">
+            <td class="header" v-if="isNameRequest(item.corpType.code)">
+              <div v-for="(name, i) in item.nameRequest.names" :key="`nrName: ${i}`" class="pb-1 names-block">
+                <v-icon v-if="isRejectedName(name)" color="red" class="names-text pr-1">mdi-close</v-icon>
+                <v-icon v-if="isApprovedName(name)" color="green" class="names-text pr-1">mdi-check</v-icon>
+                <strong class="names-text">{{ name.name }}</strong><br>
+              </div>
+            </td>
+            <td class="header" v-else>
+              <strong>{{ name(item) }}</strong>
+            </td>
+            <td v-if="showCol(headers[1].text)">{{ number(item) }}</td>
+            <td v-if="showCol(headers[2].text)" class="type-col">
+              <span class="header"><strong>{{ type(item) }}</strong></span><br>
+              <span v-if="isNameRequest(item.corpType.code)">{{ typeDescription(item.nameRequest.legalType) }}</span>
+            </td>
+            <td v-if="showCol(headers[3].text)">{{ status(item) }}</td>
+            <td v-if="showCol(headers[4].text) && isPremiumAccount">{{ folio(item) }}</td>
```

Comment:
Is folio displayed only for premium account in new UX? Folio is something which can be present regardless of the account type.

Reply:
Currently yes, in the requirements/designs it was called out as for premium accounts only. 
Should i bring this up with our design team?

<img width="500" alt="Screen Shot 2021-08-30 at 10 17 36 AM" src="https://user-images.githubusercontent.com/53542131/131378476-95d8fd72-516a-48ba-8520-a39207c6f07d.png">